BREAKING.md

```diff
@@ -36,12 +38,25 @@ This field has been deprecated and will be removed in a future breaking change.
         fileName: string,
 -       createShareLinkType?: ShareLinkTypes,
 +       createShareLinkType?: ShareLinkTypes | ISharingLinkKind,
```

Comment:
why was this changed?

Reply:
probably because I need to sync to the latest 0 will confirm.

Reply:
Oh I think it is a bug on the  git tool